projects/packages/forms/src/blocks/contact-form/components/akismet-panel/index.js

```diff
@@ -0,0 +1,57 @@
+import { Button } from '@wordpress/components';
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import PluginIntegrationPanel from '../shared/plugin-integration-panel';
+import './styles.css';
+
+const AkismetPanel = () => {
+	const { adminUrl } = useSelect( select => {
+		const site = select( coreStore ).getEntityRecord( 'root', 'site' );
+		return {
+			adminUrl: site?.url ? `${ site.url }/wp-admin/` : null,
+		};
+	}, [] );
+
+	return (
+		<PluginIntegrationPanel
+			title={ __( 'Spam Protection', 'jetpack-forms' ) }
+			pluginSlug="akismet"
+			pluginPath="akismet/akismet"
+			installText={ __( 'Install Akismet', 'jetpack-forms' ) }
+			activateText={ __( 'Activate Akismet', 'jetpack-forms' ) }
+			description={ __(
+				"Add one-click spam protection for your forms with Akismet. Simply install the plugin and you're set.",
```

Comment:
Let's link Akismet plugin page here? https://wordpress.org/plugins/akismet/

Then they can check first what we're talking about, and plugin page seems better than Akismet.com since that's what they'll end up installing with CTA.

Reply:
Added!